cms/djangoapps/xblock_config/apps.py

```diff
@@ -0,0 +1,26 @@
+"""
+xblock_config Application Configuration
+"""
+from __future__ import absolute_import
+
+from django.apps import AppConfig
+
+import cms.lib.xblock.runtime
+import xmodule.x_module
+from openedx.core.lib.xblock_utils import xblock_local_resource_url
+
+
+class XBlockConfig(AppConfig):
+    """
+    Default configuration for the "xblock_config" Django application.
+    """
+    name = u'xblock_config'
+    verbose_name = u'XBlock Configuration'
```

Comment:
Why not `unicode_literals` above to get rid of the `u`'s?

Reply:
Or are the `u`'s needed at all?

Reply:
`unicode_literals` would work in this particular case, but [OEP-7](https://open-edx-proposals.readthedocs.io/en/latest/oep-0007.html#handling-literals-option-1-python-3-style) points out that applying it selectively to only some files in a code base can be confusing.  Some of the Python 3 working group members expressed a preference for the explicit prefix for repositories that haven't yet switched to Python 3-only.

Reply:
👍